src/storybook-components/_data/demoData.js

```diff
@@ -1591,40 +1591,14 @@ const demoData = {
             placeholder: 'Select multiple topics',
             values: [{value: 'topic_a', name: 'Topic A'}, {value: 'topic_b', name: 'Topic B'}, {value: 'topic_c', name: 'Topic C'}, {value: 'topic_d', name: 'Topic D'}]
           },
-
-          {
-            type: 'rplchecklist',
```

Comment:
This will introduce a breaking change in that version, we may keep it as deprecated so we won't have trouble to update all sites.
https://github.com/search?q=org%3Adpc-sdp+rplchecklist&type=Code

We may need to add a console warning in our Ripple form when `rplchecklist` is used. https://css-tricks.com/approaches-to-deprecating-code-in-javascript/

Reply:
This PR doesn't remove rplChecklist (it removes the story though, which I think we do want to do now so others dont use it). Good point about the deprecation message, will probably need to add to RplForm